samples/apps/smallbank/smallbank.cmake

```diff
@@ -74,7 +74,7 @@ if(BUILD_TESTS)
       NAME small_bank_sigs_forwarding
       PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/small_bank_client.py
       CLIENT_BIN ./small_bank_client
-      LABEL Small_Bank_ClientSigs_Forwarding
+      LABEL "SB sig fwd"
```

Comment:
Can we avoid spaces? The label is used for the directory name of the test in `workspace/`.

Reply:
ok